frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -227,29 +227,25 @@ export const FormComponentConfig = ({
         }
         if (rest[propertyKey].type === 'object' && rest[propertyKey].properties) {
           return (
-            <Accordion key={propertyKey}>
-              <Accordion.Item>
-                <Accordion.Header>{getComponentPropertyLabel(propertyKey, t)}</Accordion.Header>
-                <Accordion.Content>
-                  {rest[propertyKey]?.description && (
-                    <Paragraph size='small'>{rest[propertyKey].description}</Paragraph>
-                  )}
-                  <FormComponentConfig
-                    key={propertyKey}
-                    schema={rest[propertyKey]}
-                    component={component[propertyKey] || {}}
-                    handleComponentUpdate={(updatedComponent: FormComponent) => {
-                      handleComponentUpdate({
-                        ...component,
-                        [propertyKey]: updatedComponent,
-                      });
-                    }}
-                    editFormId={editFormId}
-                    hideUnsupported
-                  />
-                </Accordion.Content>
-              </Accordion.Item>
-            </Accordion>
+            <div key={propertyKey}>
+              <Label as='p'>{getComponentPropertyLabel(propertyKey, t)}</Label>
```

Comment:
I wonder if this shouldn't rather be a title here to make it look like `Andre innstillinger`, `Datamodell`, `Tekst`  🤔
I also wonder if we shouldn't remove the extra div. This would add some extra spaces between the fields 🤔 
```suggestion
            <>
              <Heading level={3} size='xxsmall'>
                {getComponentPropertyLabel(propertyKey, t)}
              </Heading>
```

#### Before (with div and with Label):
<img width="547" alt="1" src="https://github.com/Altinn/altinn-studio/assets/24462611/7c81bf2f-c045-4445-a9cc-fbf4caa1789b">

#### After (withtout div and with Heading):
<img width="547" alt="Screenshot 2024-01-23 at 19 12 42" src="https://github.com/Altinn/altinn-studio/assets/24462611/12d82a12-5a2d-4748-90dd-96c0c019b7a4">

Reply:
I agree, but I think we should keep the div to keep the ```key``` on the wrapper 😄

Reply:
I see, good catch 👍 One solution to keep the key and remove the div could be to use `React.Fragment`:
```
<React.Fragment key={propertyKey}>
```